src/state/store/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from "redux";
+import rootReducer from "../reducers/rootReducer";
+import initialState from "./initialState"
+
+const store = createStore(rootReducer, initialState);
```

Comment:
when you are creating the store like this you don't need to pass the `useDispatch` hook as an argument but in your module you can simply import the store and do `store.dispatch({type: blabkla, payload: blabla})`